tests/models/jetmoe/test_modeling_jetmoe.py

```diff
@@ -383,6 +383,15 @@ def test_past_key_values_format(self):
     def test_flash_attn_2_inference_equivalence_right_padding(self):
         self.skipTest(reason="JetMoe flash attention does not support right padding")
 
+    # Copied from tests.models.phimoe.test_modeling_phimoe.PhimoeModelTest.test_assisted_decoding_with_num_logits_to_keep with phimoe->jetmoe, Phimoe->JetMoe
+    @parameterized.expand([(None, True), ("static", False)])
+    def test_assisted_decoding_with_num_logits_to_keep(self, cache_implementation, return_legacy_cache):
+        if cache_implementation == "static":
+            self.skipTest(
+                "JetMoe doesn't support StaticCache, please check the following issue -> https://github.com/huggingface/transformers/issues/28981."
+            )
+            pass
+
```

Comment:
nit: we can skip the test in `test_utils.py` by checking if model support static cache and if cache implementation is `static`, so we don't have to replicate the same "skip" in many models

Reply:
this idea is better, i updated the code